barretenberg/cpp/src/barretenberg/aztec_ivc/aztec_ivc.test.cpp

```diff
@@ -132,6 +132,24 @@ TEST_F(AztecIVCTests, Basic)
     EXPECT_TRUE(ivc.prove_and_verify());
 };
 
+/**
+ * @brief A simple test demonstrating IVC for four mock circuits, which is slightly more than minimal.
+ * @details When accumulating only four circuits, we execute all the functionality of a full AztecIVC run.
+ *
+ */
+TEST_F(AztecIVCTests, BasicFour)
```

Comment:
could disable for CI given we have n=2 and n=6 already.

Reply:
why even add this? this case is already contained in the failure test below

Reply:
I'm using it to measure memory since it's the minimal case that does everything. Yeah I guess the failure test does 4 circuits but it was a failure test I guess. 

I can just disable this and leave a comment saying it's for memory benchmarking

Reply:
oh I see, didn't realize it was the memory case

Reply:
actually, I'll leave it because it's faster for iteration than the 6 case.